packages/zlib/zlib.0.6/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Christopher Zimmermann <christopher@gmerlin.de>"
+authors: "Christopher Zimmermann <christopher@gmerlin.de>"
+homepage: "https://github.com/madroach/ocaml-zlib"
+dev-repo: "git+https://github.com/madroach/ocaml-zlib.git"
+bug-reports: "https://github.com/madroach/ocaml-zlib/issues"
```

Comment:
something that might be useful for the next version... if you add a field:

```
doc: "https://madroach.github.io/ocaml-zlib/"
```

and use 'dune-release tag && dune-release' to release this, it will also upload the `odoc` documentation of the library to the gh-pages branch of the repository.